repair/row_level.cc

```diff
@@ -3169,6 +3184,9 @@ class row_level_repair {
                 master.set_repair_state(repair_state::row_level_stop_finished, node);
             })).get();
 
+            // wait for view update to complete
+            master.wait_for_view_update_done().get();
```

Comment:
This only waits for the view update on the master node which means view updates on the repair peer nodes are not waited for.